### ISSUE TITLE
E3001 Invalid or unsupported Type AWS::GuardDuty::Member in eu-west-3

### ISSUE THREAD

**Issue by calebplum** [aws-cloudformation/cfn-python-lint#1838]
*cfn-lint version: cfn-lint 0.44.0*

*Description of issue.*
Error `E3001 Invalid or unsupported Type AWS::GuardDuty::Member in eu-west-3` when deploying a template containing an `AWS::GuardDuty::Member` resource in the `eu-west-3` region.

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.
```
---

AWSTemplateFormatVersion: 2010-09-09
Description: Deploys an AWS GuardDuty master account and invites member accounts

Resources:
  MasterDetector:
    Type: AWS::GuardDuty::Detector
    Properties:
      Enable: True

  GuardDutyMember:
    Type: AWS::GuardDuty::Member
    Properties:
      DetectorId: !Ref MasterDetector
      DisableEmailNotification: True
      Email: {member_account_email}
      MemberId: '{member_account_number}'
      Status: Invited
```

  * If present, please add links to the (official) documentation for clarification.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-guardduty-member.html
  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
Confirmed that issue persists after updating cfn-lint to 0.44.0 and running `cfn-lint --update-specs`. Confirmed template deploys successfully without linting.

**Comment by PatMyron** [comment]
**Can workaround with [resource-level ignores](https://github.com/aws-cloudformation/cfn-python-lint#resource-based-metadata) until this fix is released:**

```yaml
  Resource:
    Type: AWS::GuardDuty::Member
    Metadata:
      cfn-lint:
        config:
          ignore_checks:
          - E3001
```
